newIDE/app/src/ProjectManager/EventFunctionExtensionNameVerifier.js

```diff
@@ -0,0 +1,10 @@
+//@flow
+export function isExtensionNameTaken(
+  name: string,
+  project: gdProject
+): boolean {
+  for (let extensionName of project.getUsedExtensions().toJSArray()) {
```

Comment:
As we talked in another PR, getUsedExtensions is not containing all the extensions. In particular, all the extensions implemented in JS are not there, and we should avoid this function at all price for now.

I think we should instead check the names of all the `gd.JsPlatform.get().getAllPlatformExtensions()` :) 
Note that these extensions will also include the extensions generated from the "EventsFunctionExtension" of the project... which is ok because it will just be redundant with `project.hasEventsFunctionsExtensionNamed` (both are needed though, if you just added a events functions extension to your project, it's not yet generated and loaded as a real extension in the JsPlatform. So you need both to guard against the loaded extension names and the `hasEventsFunctionsExtensionNamed `).

Reply:
oh, yes, right I forgot about that 😓I'll change that right away